compiler/include/driver.h

```diff
@@ -304,6 +304,15 @@ extern const char* compileCommandFilename;
 extern const char* compileCommand;
 extern char compileVersion[64];
 
+// This is where the list of all supported editions goes.  Be sure to update
+// setEdition when this changes!
+extern std::string editions[2];
```

Comment:
I'm belatedly realizing I could be using a list instead of an array here.  Why that didn't occur to me at any other point in the last few weeks of sporadically poking at this in spare cycles, I'll never know 😅

Reply:
Ended up realizing a vector was even better, so used that